v1alpha/store/store.proto

```diff
@@ -15,6 +15,7 @@ message GraphItem {
     bytes k2 = 3;
     GraphItemEncoding encoding = 4;
     bytes graphItemData = 5;
+    bytes k3 = 6;
```

Comment:
This is the change here, others are generated files for this change

Reply:
I'm OK with re-numbering these fields. Lets also update encoding and graphItemData to start at 5 instead of having them be completely sequential.

Reply:
Re-numbering could have adverse effects, as per the doc. But because we're using this service internally, I guess we're okay?

And yes, it makes sense on reserving some numbers and being `encoding` at 5

Reply:
It is a breaking change when data is serialized / deserialized using the proto format. Right now, these structs are all used in memory and never serialized / deserialized as a proto over a wire or when stored.

Reply:
Hmm when a gRPC client requests data from a gRPC service (in this case Tracker to GraphStore), is data not serialized/deserialized? I suppose, we'll see a problem if Tracker uses a old version of the generated code and GraphStore serves using the new schema?

Reply:
Any way, are we re-numbering? I'm confused because the PR is merged without that. I can quickly open another, if we are renumbering.

Reply:
In this specific case, the graphstore service isn't exposed as a grpc service people can dial.
https://github.com/deps-cloud/tracker/blob/master/main.go#L39

As for the renumbering... woops. I must have misread something yesterday when working through my backlog. Feel free to put up another PR and I'll merge it in. Sorry about that.